testsys/src/install.rs

```diff
@@ -0,0 +1,33 @@
+use crate::error::{IntoError, Result};
+use clap::Parser;
+use model::test_manager::{ImageConfig, TestManager};
+
+/// The install subcommand is responsible for putting all of the necessary components for testsys in
+/// a k8s cluster.
+#[derive(Debug, Parser)]
+pub(crate) struct Install {
+    /// Controller image pull secret
+    #[clap(long = "controller-image-secret", short = 's')]
+    secret: Option<String>,
+
+    /// Controller image uri
+    // Todo add default controller_uri after images are published.
```

Comment:
nit: follows the TODO syntax elsewhere and that IDEs tend to recognize

```suggestion
    // TODO - add default controller_uri after images are published.
```